arrow-core-data/src/main/kotlin/arrow/typeclasses/Foldable.kt

```diff
@@ -205,22 +205,23 @@ interface Foldable<F> {
   /**
    * Get the first element of the foldable or none
    */
-  fun <A> Kind<F, A>.firstOption(): Option<A> = get(0)
+  fun <A> Kind<F, A>.firstOption(): Option<A> =
+    find { true }
 
   /**
    * Get the first element of the foldable or none if empty or the predicate does not match
    */
```

Comment:
We should probably change this text

Reply:
Something like..
_Get the first element matching the predicate or none_

Reply:
Sounds good